refinery/core/views.py

```diff
@@ -693,13 +696,80 @@ def graph(self, request, *args, **kwargs):
         )
 
 
-class NodeViewSet(viewsets.ModelViewSet):
-    """API endpoint that allows Nodes to be viewed"""
-    queryset = Node.objects.all()
-    serializer_class = NodeSerializer
-    lookup_field = 'uuid'
-    http_method_names = ['get']
-    # permission_classes = (IsAuthenticated,)
+class NodeViewSet(APIView):
+    """API endpoint that allows Nodes to be viewed".
+     ---
+    #YAML
+
+    PATCH:
+        parameters_strategy:
+        form: replace
+        query: merge
+
+        parameters:
+            - name: uuid
+              description: User profile uuid used as an identifier
+              type: string
+              paramType: path
+              required: true
+            - name: file_uuid
+              description: uuid for file store item
+              type: string
+              paramType: form
+              required: false
+    ...
+    """
+    http_method_names = ['get', 'patch']
+
+    def update_file_store(self, old_uuid, new_uuid):
+        if new_uuid is None:
```

Comment:
Even though the api is only supporting file_uuid deletion, this is needed to avoid future issues when the file_uuid is not changed and other fields are.